### DISCUSSION TITLE
WHY is this my most active repo?

### DISCUSSION THREAD

**Discussion by afkvido** [gemvido/old-afkvido-website#32] [Forum]
bruh

**Comment by donaldli2020** [comment]
idk

**Reply by afkvido** [reply]
bruh

**Comment by afkvido** [comment]
Probably because it's just HTML and other website stuff. Java requires major learning, however. That's probably why people go here instead of my Java projects.

smh

**Reply by donaldli2020** [reply]
html is like the simplest language
